core/block/import/markdown/anymark/testdata/html_cases.json

```diff
@@ -121,10 +121,15 @@
     "html": "<p>i<b>p</b>s  <i>um</i>  ololo</p>",
     "blocks": [{"id":"1","Content":{"text":{"text":"ips um ololo","marks":{"marks":[{"range":{"from":1,"to":2},"type":3},{"range":{"from":4,"to":6},"type":2}]}}}}]
   },
+  {
+    "desc": "image inside bold",
+    "html": "<b><img title=\"Placeholder image\" src=\"https://lh3.googleusercontent.com/123\"/></b>",
+    "blocks":  [{"id":"1","Content":{"file":{"name":"https://lh3.googleusercontent.com/123","type":2}}}]
```

Comment:
This link was killed by wrong parsing (it parsed like https://lh3.googleusercontent.com/123**)